RELEASE.md

```diff
@@ -0,0 +1,14 @@
+Release type: minor
```

Comment:
Won't it be braking change for people who already inherit from  `GraphQLView` and expecting only one required argument in  `.get_context(request)` method?

Reply:
yes, it is breaking 😊 we are still pre 1.0 so it's fine to just bump to minor :)